ddprof-lib/src/main/cpp/callTraceStorage.cpp

```diff
@@ -50,7 +50,9 @@ class LongHashTable {
         if (table != NULL) {
             table->_prev = prev;
             table->_capacity = capacity;
-            table->_size = 0;
+            // The reset is not useful with the anon mmap setting the memory is zeroed.
+            // However this silences a false positive and should not have a performance impact.
+            table->clear();
```

Comment:
I could make this a conditional change. I do not think it matters in terms of performance, as it is a memset at startup and on growth.

Reply:
Considering that the next sub-table is 2x size of the previous one the number of memset calls should not get high.